src/interpreter/util/node/get-inner-node.ts

```diff
@@ -1,12 +1,11 @@
-import { TS } from "../../../type/ts.js";
+import {TS} from "../../../type/ts.js";
 
 export function getInnerNode<T extends TS.Node>(node: TS.Node, typescript: typeof TS): T {
 	if (typescript.isParenthesizedExpression(node)) return getInnerNode(node.expression, typescript);
 	else if (typescript.isAsExpression(node)) return getInnerNode(node.expression, typescript);
-	else if (typescript.isTypeAssertionExpression?.(node) || (!("isTypeAssertionExpression" in typescript) && (typescript as typeof TS).isTypeAssertion(node))) {
-		return getInnerNode(node.expression, typescript);
-	}
-	else {
+	else if (typescript.isTypeAssertionExpression?.(node) || (!("isTypeAssertionExpression" in typescript) && (typescript as any).isTypeAssertion(node))) {
```

Comment:
Same feedback as the other two places where we check for a TypeAssertionExpression.